### DISCUSSION TITLE
Adding Laravel Policy to the File Manager

### DISCUSSION THREAD

**Discussion by codel00ps** [oneduo/nova-file-manager#423] [Q&A]
Hello, I have installed and used the file manager tool, however I was not able to add a laravel policy to prevent users based on permissions. How is that possible?

**Comment by Rezrazi** [comment]
The package has access control features built into it.
Perhaps it can be coupled to Laravel's Gate authorization system.

https://oneduo.github.io/nova-file-manager/access-control.html
